apps/etterlatte-saksbehandling-ui/client/src/components/klage/vurdering/KlageVurderingForms.tsx

```diff
@@ -0,0 +1,208 @@
+import {
+  AARSAKER_OMGJOERING,
+  InnstillingTilKabalUtenBrev,
+  Klage,
+  KlageUtfallUtenBrev,
+  LOVHJEMLER_BP,
+  LOVHJEMLER_OMS,
+  Omgjoering,
+  TEKSTER_AARSAK_OMGJOERING,
+  TEKSTER_LOVHJEMLER,
+  Utfall,
+} from '~shared/types/Klage'
+import React from 'react'
+import { Heading, Select, Textarea } from '@navikt/ds-react'
+import { FieldOrNull } from '~shared/types/util'
+import { Control, Controller } from 'react-hook-form'
+import { Feilmelding, VurderingWrapper } from '~components/klage/styled'
+import { useKlage } from '~components/klage/useKlage'
+import { SakType } from '~shared/types/sak'
+
+export type FilledFormDataVurdering = {
+  utfall: Utfall
+  omgjoering?: Omgjoering
+  innstilling?: InnstillingTilKabalUtenBrev
+}
+
+export type FormdataVurdering = FieldOrNull<FilledFormDataVurdering>
+
+export function mapFraFormdataTilKlageUtfall(skjema: FilledFormDataVurdering): KlageUtfallUtenBrev {
+  switch (skjema.utfall) {
+    case Utfall.DELVIS_OMGJOERING:
+      return { utfall: 'DELVIS_OMGJOERING', omgjoering: skjema.omgjoering!!, innstilling: skjema.innstilling!! }
+    case Utfall.OMGJOERING:
+      return { utfall: 'OMGJOERING', omgjoering: skjema.omgjoering!! }
+    case Utfall.STADFESTE_VEDTAK:
+      return { utfall: 'STADFESTE_VEDTAK', innstilling: skjema.innstilling!! }
+    case Utfall.AVVIST_MED_OMGJOERING:
+      return { utfall: 'AVVIST_MED_OMGJOERING', omgjoering: skjema.omgjoering!! }
+    case Utfall.AVVIST_MED_VEDTAKSBREV:
+      return { utfall: 'AVVIST_MED_VEDTAKSBREV' }
+  }
+}
+
+export function mapKlageTilFormdata(klage: Klage | null): FormdataVurdering {
+  if (!klage || !klage.utfall) {
+    return { utfall: null, omgjoering: null, innstilling: null }
+  }
+  const utfall = klage.utfall
+  switch (utfall.utfall) {
```

Comment:
Siden denne er felles trenger den kanskje å håndtere switch mellom de to "hovedstiene" hvis man har gått tilbake og endret om klagefrist er overholdt. Eks hvis jeg velger overholdt, legger inn noe i innstilling og har det som utfall, og så endrer formkrav til klagefrist ikke overholdt vil mappingen i det jeg får ut bli rar. Men dette kan evt. fikses i backend når vi lagrer formkrav